Hypnotic-Herons/Sprint-2
# MK and MD added trainingprograms_list.html

##Added 
trainingprograms_list.html

##Steps to Test
git fetch --all
git checkout mk_md_training_programs
code .
python manage.py makemigrations HRIT
python manage.py migrate
python manage.py createsuperuser (this is just for your local use)
login on the admin view http://127.0.0.1:8000/admin
add a few training programs
look at the browser view for training programs to see if they are there http://127.0.0.1:8000/HRIT/trainingprograms
you should see the training programs in a list
